ci/workflows/main.yml

```
@@ -44,6 +44,23 @@ steps:
     timeout_in_minutes: 20
     retry: *auto-retry
 
+  - label: "build other components"
+    command: "ci/scripts/build-other.sh -t ci-dev -p ci-dev"
```

I guess this is unused? Same for other workflows.

```suggestion
    command: "ci/scripts/build-other.sh"
```